crates/biome_cli/src/execute/traverse.rs

```diff
@@ -537,7 +537,7 @@ impl TraversalContext for TraversalOptions<'_, '_> {
             //   Note that `symlink/subdir` is not an existing file.
             let can_handle = !self
                 .workspace
-                .is_path_ignored(IsPathIgnoredParams {
+                .is_path_ignored(PathIsIgnoredParams {
```

Comment:
I had to rename this type because the initial `I` was in conflict with the WASM types, which all start with `I`